docs/cheaha/migration_overview.md

```diff
@@ -0,0 +1,194 @@
+    1. After migration, each account will require certification by going to our [web portal](https://rc.uab.edu). You will not be able to access Cheaha until your account has been certified.
+
+1. **Migration Schedule**
+    1. Plan to lose access to Cheaha for up to **2 days** during migration.
+    1. If migration for a community completes earlier than expected, the next community will immediately receive a notice for pending migration starting their 2 day premigration clock.
+        1. Due to potential for high variability in migration duration for each community, we cannot provide set dates for each community's migration. The migration will be performed as safely and quickly as possible.
+    1. Migrations will continue as possible over all necessary weekends.
+
+1. **Post-Migration Cleanup**
+    1. After migration for a batch ends, held jobs will be released.
+    1. Compute node availability differs between GPFS 4 and GPFS 5. See [below](#gpfs-5-compute-nodes) for more details.
+
+### General Timeline
+
+As explained above, the migration can be thought of as a series of, at most, 4-day windows. Each window will see one or more community migration(s) performed from start to finish.
+
+1. Day 0: Community members receive an email notifying them of their impending migration
+1. Day 2: Community members are put on hold and lose access to Cheaha
+1. Day 2-4: Access to Cheaha is restored post-migration
+1. Repeat for the next community
+
+Due to high variability in file structure and content across users and communities, it's not feasible to predict how long any given community's migration will take. Therefore, we cannot give expected dates for when a given group's migration will begin and end as it's solely dependent on migrations of prior groups. Here are some dates to remember though:
+
+- **Monday, October 6**: Initial announcement concerning the migration is sent to the Cheaha userbase
+- **Wednesday, October 8**: Notification of impending migration sent to members of community 1
+- **Friday, October 10**: Migration for community 1 begins
+
+## GPFS 5 Data Tiering
+
+With the upgrade to GPFS 5, we are introducing a data tiering strategy to help us balance increasing storage demands with rising costs for high performance storage systems. GPFS will still be used as the primary parallel performance storage system and will store all files smaller than 4 MiB and all other active files. Inactive files will be transparently moved to a CephFS storage tier to reduce system pressure on GPFS. Inactivity will be determined by time since last file access. Files stored on CephFS leave a stub behind on GPFS. The stub contains all relevant metadata for the file and will appear to the user as if the file is still on GPFS. Reading a file stub will cause the file to be automatically transferred from CephFS back to GPFS.
```

Comment:
Swap out CephFS for "Core Ceph", we're on a path to use that as our "brand" name, I believe. CephFS can be used if they need to know more about the underlying infra.

Reply:
CephFS (to me at least) implies a file system from the FS in the name. I don't expect most people to realize that, but there will be some who do. For those people, they would most likely be able to visualize a secondary file system "mirroring" GPFS and better understand exactly how it's being used to store their data. 

I think Core Ceph is too abstract of a name for anyone to figure out exactly how this backend maps to GPFS in terms of storing their data. The vast majority of people won't care or ask, but the ones who do, I think, would have their questions answered by knowing it's essentially just another file system "mirroring" GPFS.

Caveat: I know Ceph doesn't mirror GPFS. I couldn't think of a better description for how I'm thinking of it